restcomm/restcomm.telephony/resources/mybatis.xml

```diff
@@ -19,26 +19,28 @@
   
   
   <mappers>
-    <mapper resource="mappers/accounts.xml"/>
-    <mapper resource="mappers/applications.xml"/>
-    <mapper resource="mappers/available-phone-numbers.xml"/>
-    <mapper resource="mappers/call-detail-records.xml"/>
-    <mapper resource="mappers/conference-detail-records.xml"/>
-    <mapper resource="mappers/clients.xml"/>
-    <mapper resource="mappers/http-cookies.xml"/>
-    <mapper resource="mappers/incoming-phone-numbers.xml"/>
-    <mapper resource="mappers/notifications.xml"/>
-    <mapper resource="mappers/outgoing-caller-ids.xml"/>
-    <mapper resource="mappers/registrations.xml"/>
-    <mapper resource="mappers/recordings.xml"/>
-    <mapper resource="mappers/sand-boxes.xml"/>
-    <mapper resource="mappers/short-codes.xml"/>
-    <mapper resource="mappers/sms-messages.xml"/>
-    <mapper resource="mappers/transcriptions.xml"/>
-    <mapper resource="mappers/gateways.xml"/>
-    <mapper resource="mappers/announcements.xml"/>
-    <mapper resource="mappers/extensions-configuration.xml"/>
-    <mapper resource="mappers/media-servers.xml"/>
-    <mapper resource="mappers/media-resource-broker-entity.xml"/>
+    <mapper resource="org/restcomm/connect/dao/mybatis/accounts.xml"/>
```

Comment:
@jaimecasero why you point to the dao resources mappers? Those are also copies of the proper mappers that are located to:
restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/sql
and
restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/sql

I believe when you apply the fix for HSQL to have common mappers, mappers should be moveds to the resources of the restcomm.application and all tests should point to them as you said back then

Reply:
@jaimecasero my comment here was just a suggestion for how to get rid the various mapper files as you suggested.
The mybatis.xml here is used for testing.
Not important now, it was just a comment

Reply:
i completely removed this unused dir,and the pom part as discussed.